content/02.introduction.md

```diff
@@ -1,29 +1,25 @@
 ## Introduction
+
+Scientific communication has been around since the beginning of science where the main form of communication was through scientific meetings [@raw:smith2006trouble].
+Communicating through scientific journals started back in the 17th century [@raw:smith2006trouble] but didn't pick up until the 1960s where National Institutes of Health (NIH) created an exchange network that allowed scientists to share unreferred manuscripts with one another [@doi:10.1371/journal.pbio.2003995].
+These networks were brough to a halt due to pressure from scientific journals; however, this sparked a movement to begin using preprints within the scientific community [@doi:10.1371/journal.pbio.2003995]. 
+The longest recorded use of preprints was with physicists during the 1990s [@doi:10.15252/embj.201695531] and over the past few decades preprints have made a resurgence within the life sciences community [@doi:10.7554/eLife.45133.001' @doi:10.1038/530265a; @doi:10.1016/j.bpj.2016.06.035; @doi:10.1126/science.aaf9133; @doi:10.1038/s41557-020-0477-5; @doi:10.1371/journal.pmed.1002549; @doi:10.1371/journal.pbio.3000151; @doi:10.1099/mgen.0.000259].

```

Comment:
To Larry's point, I wonder if these articles are about preprints or preprint repositories:

> The changes recently really are as much about search and free online access (characteristics of repositories, not preprints themselves) as much as it is about scientists writing them or making them available to peers. This distinction actually is worth making throughout.

For example, does that paper about physics in the 90s actually discuss arXiv (a repository) as opposed to the use of preprints? If so, this needs to be restructured a bit since you draw a distinction below.